Description: Fix info navigation structure
 In the original when using the standalone 'info' program pressing
 spacebar does not navigate through the pages.  This patch
 restructures the node navigation somewhat to enable this to happen.
 Reported by Russ Allbery.
Author: Bob Proulx

--- a/time.texi
+++ b/time.texi
@@ -70,7 +70,10 @@
 by the Foundation.
 @end titlepage
 
-@node Top, , (dir), (dir)
+@contents
+
+@node Top
+@top The GNU @code{time} Command
 
 @ifinfo
 This file documents the the GNU @code{time} command for running programs
@@ -80,30 +83,12 @@
 
 @menu
 * Resource Measurement::  Measuring program resource use.
-
- --- The Detailed Node Listing ---
-
-Measuring Program Resource Use
-
-* Setting Format::      Selecting the information reported by @code{time}.
-* Format String::	The information @code{time} can report.
-* Redirecting::         Writing the information to a file.
-* Examples::		Examples of using @code{time}.
-* Accuracy::		Limitations on the accuracy of @code{time} output.
-* Invoking time::	Summary of the options to the @code{time} command.
-
-The Format String
-
-* Time Resources::
-* Memory Resources::
-* I/O Resources::
-* Command Info::
+* Concept index::  Index of concepts.
 @end menu
 
-@node Resource Measurement, , , Top
+@node Resource Measurement
 @chapter Measuring Program Resource Use
-@findex time
-@kindex time
+@cindex time invocation
 @pindex time
 @pindex measurement
 
@@ -125,7 +110,7 @@
 time @r{[}option@dots{}@r{]} @var{command} @r{[}@var{arg}@dots{}@r{]}
 @end example
 
-@kindex resources
+@cindex resource specifiers
 @code{time} runs the program @var{command}, with any given arguments
 @var{arg}@dots{}.  When @var{command} finishes, @code{time} displays
 information about resources used by @var{command}.
@@ -197,7 +182,7 @@
 
 @item -v
 @itemx --verbose
-@cindex format
+@cindex verbose format
 Use the built-in verbose format, which displays each available piece of
 information on the program's resource use on its own line, with an
 English description of its meaning.
@@ -207,7 +192,6 @@
 @section The Format String
 
 @cindex format
-@kindex resource
 The @dfn{format string} controls the contents of the @code{time} output.
 It consists of @dfn{resource specifiers} and @dfn{escapes}, interspersed
 with plain text.
@@ -450,7 +434,7 @@
 
 @node Accuracy
 @section Accuracy
-@kindex error (in measurement)
+@cindex error (in measurement)
 
 The elapsed time is not collected atomically with the execution of the
 program; as a result, in bizarre circumstances (if the @code{time}
@@ -489,7 +473,7 @@
 time @r{[}option@dots{}@r{]} @var{command} @r{[}@var{arg}@dots{}@r{]}
 @end example
 
-@kindex resources
+@cindex resources
 @code{time} runs the program @var{command}, with any given arguments
 @var{arg}@dots{}.  When @var{command} finishes, @code{time} displays
 information about resources used by @var{command} (on the standard error
@@ -524,7 +508,7 @@
 
 @item -v
 @itemx --verbose
-@cindex format
+@cindex verbose option
 Use the built-in verbose format.
 
 @item -V
@@ -533,5 +517,9 @@
 Print the version number of @code{time} and exit.
 @end table
 
-@contents
+@node Concept index
+@unnumbered Concept index
+
+@printindex cp
+
 @bye
